tests/test_search.py

```diff
@@ -1490,6 +1472,23 @@ def test_searchparam_ignored_pretty_format(es_data, engine):
     assert bundle.total == 1
 
 
+def test_searchparam_elements(es_data, engine):
```

Comment:
can you test the case where you specify an _element which is not an attribute of the resource ?

Reply:
Not sure what the behavior for this should be... Should we ignore the element or return an OperationOutcome? (I don't think this is specified in the spec)